pkg/master/master.go

```diff
@@ -781,11 +782,14 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
 	controllerStorage := expcontrolleretcd.NewStorage(c.DatabaseStorage)
 	autoscalerStorage := horizontalpodautoscaleretcd.NewREST(c.DatabaseStorage)
+	nodeMetricsStorage := nodemetrics.NewREST(c.DatabaseStorage)
 
 	storage := map[string]rest.Storage{
 		strings.ToLower("replicationControllers"):       controllerStorage.ReplicationController,
 		strings.ToLower("replicationControllers/scale"): controllerStorage.Scale,
 		strings.ToLower("horizontalpodautoscalers"):     autoscalerStorage,
+		// /api/experimental/nodeMetrics/{myNode}
```

Comment:
The inconsistency between the various map keys here is bad.  Every key should be in "camelCase" with to lower around it.  So `horizontalPodAutoscalers` and `nodeMetrics`